model.md

```diff
@@ -0,0 +1,4 @@
+The *verification* problem can be solved by cryptographic signing (implemented in IPFS)  
+The *authority* problem can be solved by webs of trust (to be implemented in Data Together app)  
+The *discovery* problem can be solved by ??? (archivers has a “coverage” service that might be a model, but ultimately this should be decentralized, right?)  
```

Comment:
Yeah, I'm interested in this too. Right now, the DT interface seems to offer very limited search tools. But eventually we probably want users to at least be able to search through the set of entitites that have been processed by the DT app. This, of course, is a pretty centralized model of search! Decentralizing that function would be... togugh, right?  Maybe @b5 has some thoughts on that (which I guess would be a very late stage of development).